### ISSUE TITLE
Implement GitHub hook

### ISSUE THREAD

**Issue by nateabele** [radify/PR.js#7]
https://github.com/settings/applications/206639

**Comment by eddiemonge** [comment]
dead link. is this so a bot or something can comment back on the issue?

**Comment by jfmercer** [comment]
Nah, @nateabele's account is not a bot. But I do wonder what the original intent of this issue is.

**Comment by nateabele** [comment]
> dead link. is this so a bot or something can comment back on the issue?

Yeah, so it can use the Build Status API. And yeah, sorry, that link was for my reference. It's an app settings page on my account.
